week-3/D-array-filter/exercise.js

```diff
@@ -5,9 +5,30 @@
   - Finish the statement on line 10 to produce an array with valid content
   - Do not edit any of the other existing code
 */
+function valuesBreaksYourCode(element) {
```

Comment:
The function name here implies that the value breaks the code. In fact it is checking if the value is valid, so it would be better named `isValidPair`.